cli/package.json

```diff
@@ -22,6 +22,14 @@
   "bin": {
     "create-t3-app": "./dist/index.js"
   },
+  "files": [
+    "dist",
+    "template",
+    "README.md",
+    "CHANGELOG.md",
+    ".yarnrc.yml",
+    "package.json"
+  ],
```

Comment:
not related to app-dir change, just wanting to minimize the bundle size since this PR adds a lot of new template files